dts/x86/intel_quark_d2000.dtsi

```diff
@@ -0,0 +1,57 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		cpu@0 {
+			compatible = "intel,quark";
```

Comment:
is the cpu core the same as on quark_se_c1000?  Is there really only one quark cpu core?